metrics-exporters/README.md

```diff
@@ -1,6 +1,6 @@
 # Metrics Exporters (in development)
 
-Before deploying, you must export your GITHUB_USER and GITHUB_TOKEN to your shell. Then run:
+Before deploying, you must export your GITHUB_REPOS, GITHUB_USER and GITHUB_TOKEN to your shell. Then run:
```

Comment:
Should we add an example to make it obvious, since GITHUB_REPOS is a comma separated list of items in the form of "user/repo"